packages/desktop-client/src/components/spreadsheet/useSheetValue.js

```diff
@@ -1,47 +1,19 @@
-import { useContext, useState, useRef, useLayoutEffect } from 'react';
+import { useState, useRef, useLayoutEffect } from 'react';
 
 import { useSpreadsheet } from 'loot-core/src/client/SpreadsheetProvider';
 
-import NamespaceContext from './NamespaceContext';
-
-function unresolveName(name) {
-  let idx = name.indexOf('!');
-  if (idx !== -1) {
-    return {
-      sheet: name.slice(0, idx),
-      name: name.slice(idx + 1),
-    };
-  }
-  return { sheet: null, name };
-}
+import useSheetName from './useSheetName';
 
 export default function useSheetValue(binding, onChange) {
   if (!binding) {
-    throw new Error('SheetValue binding is required');
-  }
-  if (global.IS_TESTING && typeof binding !== 'string' && !binding.name) {
-    binding = { ...binding, name: binding.value.toString() };
+    throw new Error('Binding is required');
```

Comment:
Can’t you skip this since `useSheetName` does the check already?

Reply:
Removing this :)